pom.xml

```diff
@@ -320,9 +320,8 @@
         <clojure.test.set>!integration.*</clojure.test.set>
         <skipITs>true</skipITs>
 
-        <aetherVersion>1.1.0</aetherVersion>
-        <mavenVersion>3.1.0</mavenVersion>
-        <wagonVersion>1.0</wagonVersion>
+        <maven-resolver.version>1.3.3</maven-resolver.version>
+        <maven.version>3.6.0</maven.version>
```

Comment:
This might be a bit out of scope, but maybe we should use this variable in the enforcer plugin below, so we require at least this Maven version?

Reply:
Maybe we don't need to align maven version for build and maven version for storm-submit-tool. The propery has been only used for storm-submit-tool.

Reply:
Ok. It probably doesn't matter then.